lib/Marlin/Marlin/src/module/prusa/toolchanger_utils.cpp

```diff
@@ -418,9 +418,9 @@ bool PrusaToolChangerUtils::load_tool_offsets_from_usb() {
 }
 
 const PrusaToolInfo &PrusaToolChangerUtils::get_tool_info(const Dwarf &dwarf, bool check_calibrated) const {
-    assert(dwarf.get_dwarf_nr() <= tool_info.size());
-    assert(dwarf.get_dwarf_nr() > 0);
-    const PrusaToolInfo &info = tool_info[dwarf.get_dwarf_nr() - 1];
+    assert(dwarf.get_dwarf_nr() < tool_info.size());
+    assert(dwarf.get_dwarf_nr() >= 0);
```

Comment:
This will be always true, dwarf_nr is uint8_t

Reply:
Agree.